### PR TITLE
Add documentation for SKR tests

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

Changes proposed in this pull request:

- Document github actions ESLint workflow
- Document SKR tests
- Remove old SKR documentation

**Related issue(s)**

#3

### CODE REVIEW

**Review by PK85 — APPROVED** [review]

File: docs/contributor/05-20-e2e_tests.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+### Usage
+
+The test executes the following steps:
+1. Provisions SKR on a cluster given by the customer
+2. Checks if sap-btp-manager secret exist.
+3. Deprovisions SKR on a cluster given by the customer
+
+### Test execution 
+
+1. Before you run the test, prepare the `.env` file based on the following [.env.template](/testing/e2e/skr/skr-test/.env.template):
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:70]:
```suggestion
1. Before you run the test, prepare the `.env` file based on the following [`.env.template`](/testing/e2e/skr/skr-test/.env.template):
```

File: docs/contributor/05-20-e2e_tests.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+## E2E SKR AWS upgrade integration test
+
+### Usage
+
+The test executes the following steps:
+1. Provisions an SKR cluster.
+2. Runs Kyma upgrade.
+3. Deprovisions the SKR instance and clean up the resources.
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:89]:
```suggestion
3. Deprovisions the SKR instance and cleans up the resources.
```

File: docs/contributor/05-20-e2e_tests.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+### Test execution 
+
+1. Before you run the test, prepare the `.env` file based on the following [.env.template](/testing/e2e/skr/skr-aws-upgrade-integration/.env.template):
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:93]:
```suggestion
1. Before you run the test, prepare the `.env` file based on the following [`.env.template`](/testing/e2e/skr/skr-aws-upgrade-integration/.env.template):
```

File: docs/contributor/05-20-e2e_tests.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+### Usage
+
+The test executes the following steps:
+1. Calls KEB endpoints without an authorization token.
+2. Checks whether the call was rejected.
+
+### Test execution 
+
+1. Before you run the test, prepare the `.env` file based on the following [.env.template](/testing/e2e/skr/keb-endpoints-test/.env.template):
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:115]:
```suggestion
1. Before you run the test, prepare the `.env` file based on the following [`.env.template`](/testing/e2e/skr/keb-endpoints-test/.env.template):
```

File: docs/contributor/05-20-e2e_tests.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+3. Run the test scenario.
+```bash
+make keb-endpoints
+```
+
+### CI pipelines
+
+The tests are run once per day at 01:05 by the given prowjobs:
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:129]:
```suggestion
The tests are run once per day at 01:05 by the given Prow jobs:
```

File: docs/contributor/05-20-e2e_tests.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
+# E2E tests of kyma-environment-broker
```
[the remark this answers is not in the document]
Comment (MarekMichali) [reply docs/contributor/05-20-e2e_tests.md:1]: I wanted to keep it consistent with the [btp-manager](https://github.com/kyma-project/btp-manager/blob/main/docs/contributor/05-20-e2e_tests.md#e2e-tests-of-btp-manager-installation-from-oci-module-image)

File: docs/contributor/04-10-workflows.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# GitHub Action workflows
```
[the remark this answers is not in the document]
Comment (MarekMichali) [reply docs/contributor/04-10-workflows.md:1]: So we should also change it [here?](https://github.com/kyma-project/btp-manager/blob/main/docs/contributor/04-10-workflows.md#github-action-workflows)

File: docs/contributor/05-20-e2e_tests.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+
+## Overview
+
+The end-to-end (E2E) tests covers kyma-environment-broker (KEB) and SAP BTP, Kyma runtime (SKR).
+There are four tests:
+- `skr-tests` for checking on different cloud service providers Kyma provisioning, BTP Manager secret reconcilation, updating OIDC, updating machine type and Kyma deprovisioning
```
[the remark this answers is not in the document]
Comment (MarekMichali) [reply docs/contributor/05-20-e2e_tests.md:7]: What about this:

> - `skr-tests` for testing the following operations on different cloud service providers: Kyma provisioning, BTP Manager secret reconcilation, updating OIDC, updating machine type and Kyma deprovisioning

File: docs/contributor/05-20-e2e_tests.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply docs/contributor/05-20-e2e_tests.md:1]: Let's use Kyma Environment Broker. I'll adjust the BTP Manager doc.

File: docs/contributor/05-20-e2e_tests.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply docs/contributor/05-20-e2e_tests.md:7]: Much better :)

File: docs/contributor/05-20-e2e_tests.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
+# E2E tests of Kyma Environment Broker
+
+## Overview
+
+The end-to-end (E2E) tests cover Kyma Environment Broker (KEB) and SAP BTP, Kyma runtime (SKR).
+There are four tests:
+- `skr-tests` for testing the following operations on different cloud service providers: Kyma provisioning, BTP Manager secret reconcilation, updating OIDC, updating machine type and Kyma deprovisioning
```
Comment (IwonaLanger) [inline docs/contributor/05-20-e2e_tests.md:7]:
```suggestion
- `skr-tests` for testing the following operations on different cloud service providers: Kyma provisioning, BTP Manager secret reconciliation, updating OIDC, updating machine type, and Kyma deprovisioning
```

**Review by IwonaLanger — APPROVED** [review]

**Review by PK85 — APPROVED** [review]

**Pull request merged**